common/meta/raft-store/src/config.rs

```diff
@@ -111,6 +111,12 @@ pub struct RaftConfig {
     )]
     pub single: bool,
 
+    /// Bring up a metasrv node and join a cluster.
+    ///
+    /// The value is one or more addresses of a node in the cluster, to which this node sends a `join` request.
+    #[structopt(long, env = "METASRV_JOIN")]
```

Comment:
It's would be nice to add a `help` message to tell the user how to use the `join`, I guess the need indeed.

Reply:
That's the magic! `structopt` captures the doc comment and builds help message for it! :  
```
        --join <join>...
            Bring up a metasrv node and join a cluster [env: METASRV_JOIN=]
```

Reply:
If the user inputs multiple addresses, the help message should print an input format hint, like using comma separation:)